Platform/TigerlakeBoardPkg/BoardConfig.py

```diff
@@ -216,9 +235,13 @@ def PlatformBuildHook (self, build, phase):
                 lines         = open (cfg_dlt_file).read()
 
                 # Enable TCC in dlt file
-                if self.ENABLE_TCC:
+                if self.INCLUDE_CFGDATA_TCC:
                     lines += open (os.path.join(brd_cfg_src_dir, 'CfgData_Tcc_Feature.dlt')).read()
 
+                # Enable TSN in dlt file
+                if self.INCLUDE_CFGDATA_TSN:
+                    lines += open (os.path.join(brd_cfg_src_dir, 'CfgData_Tsn_Feature.dlt')).read()

```

Comment:
Where will this file be used? If you are trying to follow CfgData_Tcc_Feature.dlt, this file is only used to create Auto generated Tcc enabled IFWIs which is done in our stitching scripts (release.py).

Reply:
So for TGLU, in addition to these enables, we will also have to stitch the IFWI with 'tsn7' cmd. Please make sure to add this option in the automated script when generating auto TSN enabled IFWI.